dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/advice/ExceptionAdvice.java

```diff
@@ -34,9 +38,14 @@ public void handleHttpMediaTypeNotAcceptableException() {}
   @ExceptionHandler(UsernameNotFoundException.class)
   public void handleNotFound() {}
 
-  @ResponseStatus(HttpStatus.UNPROCESSABLE_ENTITY)
   @ExceptionHandler(ValidationException.class)
-  public void handleValidationException(ValidationException exception) {
+  public ResponseEntity<Object> handleValidationException(ValidationException exception) {
     LOGGER.error("Validation error: ", exception);
```

Comment:
Sowas sollte maximal in den WARN-Level reinlaufen, wenn es sich um erwartete, durch den Anwender getriggerte Fehler handelt.